rhobs/alerting/data_plane/prometheus.gitops_cluster_agent_op_alerts.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-gitops-cluster-agent-operations-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: cluster-agent-operations_alerts
+    interval: 1m
+    rules:
+    - alert: GitOpsClusterAgentOperationErrors
+      expr: |
+        sum(operationDB_rows_in_non_complete_state) <= 10
```

Comment:
Should it be >= 10 or maybe from the Jira, >10 ?  eg. ......report when it exceeds a given value (e.g. 10).

Reply:
<img width="1263" alt="Screenshot 2023-11-20 at 4 22 06 PM" src="https://github.com/redhat-appstudio/o11y/assets/76523858/4dc38481-722f-4760-a091-71b35d8a9dfb">
basically what I am doing is if  $A <= 10 , if 3 <= 10 i.e true (then no alert), if 12 <= 10 i.e, false (alert), please let me know whether I should be doing this other way around?

Reply:
I think I should change this to  >=10